docs/http.md

```diff
@@ -481,6 +485,10 @@ step.
   direction of travel immediately after the maneuver.
 - `type` A string indicating the type of maneuver. **new identifiers might be introduced without API change**
    Types  unknown to the client should be handled like the `turn` type, the existance of correct `modifier` values is guranteed.
+- `lanes`: the available turn lanes at the turn
+  - `indication`: markings on the road, following the OSM scheme (e.g. left, slight_right, or through). Markings mey be combined into a list separated by ';'.
```

Comment:
> Markings mey be combined into a list separated by ';'.

I don't think this documentation is valid anymore. The Table needs to be moved up here.